### PR TITLE
feat: use stamp to resolve address to handle

### LINKED ISSUE #4055: API for fetching ENS, Lens and UD handles
### Briefly describe the feature.

Create an API with Stamp to run the logic for fetching users' handles

### Which problem is this feature trying to solve?

Currently retrieving a Lens handle sends 9 requests to Brovider and Polygon node.

### What is the expected solution?

Create an API with Stamp to handle this logic for ENS, Lens and Unstoppable Domains handles.

### Anything else you'd like to mention?

[Discord discussion](https://discord.com/channels/955773041898573854/1103266859870081114/1126717094344528005)

### LINKED ISSUE #4334: [BUG] - Lens resolver not working
### Briefly describe the bug.

lens resolver throws some weird error 

### How can we reproduce the bug?

1. Go to snapshot.org
2. connect with `0x9D422D69cbf9Aaf39A7AEC127Ef8aFd5A1cFF6a5` using guest
3. Open console to see errors 

Or proposals like this https://snapshot.org/#/polls.lenster.xyz/proposal/0xda179d73d85797602a8f788eb8a326cda541a9a23d6b6b6e785682f7a98ec93e should contain lot of .lens

### What is the expected behaviour?

should not throw errors and show .lens if it is available 

### Can you attach screenshots?

image

### Can you provide your device specific details below?

_No response_

### PR DESCRIPTION
**Pull request by wa0x6e**
### Summary

From https://github.com/snapshot-labs/pitches/issues/58

Fix #4334 
Fix #4055 

### How to test

1. ~~Wait for https://github.com/snapshot-labs/stamp/pull/113 to be merged~~
2. Go to https://snapshot-git-resolve-address-domains-using-stamp-snapshot.vercel.app/#/polls.lenster.xyz/proposal/0xda179d73d85797602a8f788eb8a326cda541a9a23d6b6b6e785682f7a98ec93e
3. It should resolves ENS and LENS addresses, with a single call to stamp API

### To-Do

- [ ]

### CODE REVIEW

File: .env [hunk 1]
```diff
@@ -4,6 +4,7 @@ VITE_SCORES_URL=https://score.snapshot.org
 VITE_ENVELOP_URL=https://core.envelop.fyi
 VITE_SIDEKICK_URL=https://sh5.co
 VITE_BROVIDER_URL=https://rpc.snapshot.org
+VOTE_STAMP_URL=https://stamp.fyi
```
Comment (samuveth) [inline .env:7]:
```suggestion
VITE_STAMP_URL=https://stamp.fyi
```

Comment (samuveth) [reply .env:7]: Should we also changes the other occurrences of stamp url? Like for avatar `https://cdn.stamp.fyi/`

Comment (wa0x6e) [reply .env]: > Should we also changes the other occurrences of stamp url? Like for avatar `https://cdn.stamp.fyi/`

We could, but it's not the same url

**Review by samuveth — APPROVED** [review]
tACK

**Pull request merged**